docs/source/policy/lambda.rst

```diff
@@ -10,80 +10,85 @@ different Amazon services can be used as event sources.
 CloudWatch Events
 #################
 
-CloudWatch Events (CWE) is a general event bus for AWS infrastructure. Currently,
-it covers several major sources of information: CloudTrail API calls
-over a poll period on CloudTrail delivery, real-time instance status
-events, autoscale group notifications, and scheduled/periodic events.
-
-CloudTrail provides a very rich data source over the entire range
-of AWS services exposed via the audit trail that allows Custodian to define effective
-realtime policies against any AWS product.
-
-Additionally, for EC2 instances we can provide mandatory policy
-compliance - this means the non-compliant resources never
-became available.
+`CloudWatch Events
+<http://docs.aws.amazon.com/AmazonCloudWatch/latest/events/WhatIsCloudWatchEvents.html>`_
+(CWE) is a general event bus for AWS infrastructure. Currently, it covers
+several major sources of information:
+
+#. CloudTrail API calls over a poll period on CloudTrail delivery,
+#. real-time instance status events,
+#. autoscale group notifications, and
+#. scheduled/periodic events.
+
+CloudTrail provides a very rich data source over the entire range of AWS
+services exposed via the audit trail that allows Custodian to define effective
+realtime policies against any AWS product. Additionally, for EC2 instances we
+can provide mandatory policy compliance - this means the non-compliant
+resources never became available.
 
 Cloud Custodian Integration
 ===========================
 
-Custodian provides for policy level execution against any CWE event
-stream. Each Custodian policy can be deployed as an independent Lambda
-function. The only difference between a Custodian policy that runs in
-Lambda and one that runs directly from the CLI in poll mode
-is the specification of the subscription of the events in the mode config block of the policy.
+Custodian provides for policy level execution against any CWE event stream.
+Each Custodian policy can be deployed as an independent Lambda function. The
+only difference between a Custodian policy that runs in Lambda and one that
+runs directly from the CLI in poll mode is the specification of the
+subscription of the events in the mode config block of the policy.
 
 Internally Custodian will reconstitute current state for all the resources
 in the event, execute the policy against them, match against the
 policy filters, and apply the policy actions to matching resources.
 
-:ref:`Mu<mu>` is the letter after Lambda, Lambda is a keyword in python.
 
-Configuration
-=============
+CloudTrail API Calls
+++++++++++++++++++++
 
-Examples
+Lambdas can receive CWE over CloudTrail API calls with a 1-15m delay.
```

Comment:
this delay information isn't correct cwe over cloudtrail delay is P99 of 90s. aws config which is a separate lambda policy target/mode has a delay 1-15m typically (sla on tag only changes is a bit higher).